erpnext/stock/onboarding_step/create_a_supplier/create_a_supplier.json

```diff
@@ -1,18 +1,19 @@
 {
- "action": "Create Entry",
+ "action": "Show Form Tour",
```

Comment:
Is Tour for Supplier missing? Or it's being worked upon in other module? @18alantom @rohitwaghchaure

Reply:
Since Supplier is present in other two modules (Buying and Accounting), it's been removed from here to avoid repetition.